build-docs.sh

```
@@ -0,0 +1,28 @@
+git clone $(pwd)/.git -b docs ./dist/fireproof-docs
+cp .git/config ./dist/fireproof-docs/.git/
+(
+  cd ./dist/fireproof-docs &&
+  git config advice.addIgnoredFile false &&
+  git pull origin docs &&
+  git rm -rf docs
+)
+mkdir -p dist/fireproof-docs/docs
+npx typedoc --out dist/fireproof-docs/docs  src/database.ts
+
+cd ./dist/fireproof-docs
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)